WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -138,233 +142,65 @@
-                    <Path Fill="{DynamicResource TextControlForeground}" Opacity="0.5"
-                          Data="m 237.18417 5 h 0.5 v 52 h -0.5 z" />
-                    <Path Fill="{DynamicResource TextControlForeground}"
-                          Data="m 264.11765 23 c 0.84465 0 1.52941 0.68474 1.52941 1.52941 v 12.94117 c 0 0.84466 -0.68476 1.52941 -1.52941 1.52941 h -2.58824 C 260.68474 38.99999 260 38.31524 260 37.47058 V 24.52941 C 260 23.68474 260.68474 23 261.52941 23 Z m 8.47058 0 c 0.84466 0 1.52941 0.68474 1.52941 1.52941 v 12.94117 c 0 0.84466 -0.68475 1.52941 -1.52941 1.52941 H 270 c -0.84466 0 -1.52942 -0.68475 -1.52942 -1.52941 V 24.52941 C 268.47058 23.68474 269.15534 23 270 23 Z m -8.47058 1.17647 h -2.58824 c -0.19492 0 -0.35294 0.15802 -0.35294 0.35294 v 12.94117 c 0 0.19492 0.15802 0.35294 0.35294 0.35294 h 2.58824 c 0.19492 0 0.35294 -0.15802 0.35294 -0.35294 V 24.52941 c 0 -0.19492 -0.15802 -0.35294 -0.35294 -0.35294 z m 8.47058 0 H 270 c -0.19492 0 -0.35295 0.15802 -0.35295 0.35294 v 12.94117 c 0 0.19492 0.15803 0.35294 0.35295 0.35294 h 2.58823 c 0.19492 0 0.35294 -0.15802 0.35294 -0.35294 V 24.52941 c 0 -0.19492 -0.15802 -0.35294 -0.35294 -0.35294 z" />
-                    <Path Fill="{DynamicResource SystemAccentColor}"
-                          Data="m 309.68417 31 a 5.8 5.8 0 0 1 -0.9 3 6 6 0 0 1 -5.1 3 5.8 5.8 0 0 1 -3 -0.9 6 6 0 0 1 -3 -5.1 5.8 5.8 0 0 1 0.9 -3 6 6 0 0 1 10.4 0.2 5.9 5.9 0 0 1 0.7 2.8 z" />
-                  </Panel>
-                </Viewbox>
-              </Panel>
-            </Border>
-          </Panel>
-        </Grid>
-        <!-- Free and Open Source -->
-        <Grid ColumnDefinitions="*,*">
-          <Viewbox Classes="icon" HorizontalAlignment="Left" VerticalAlignment="Top" Height="48" Width="48">
-            <ContentControl Template="{DynamicResource wasabi_logo_dynamic}" />
-          </Viewbox>
-          <DockPanel Grid.Column="0" Grid.Row="0" VerticalAlignment="Center">
-            <TextBlock Classes="text"
-                       Text="To ensure the code - running on your computer - can indeed protect you from the developers and the public, Wasabi is free, open-source and deterministically reproducible software." />
-            <TextBlock Text="Open-Source"
-                       Classes="title" />
+        <DockPanel LastChildFill="True" VerticalAlignment="Center">
```

Comment:
It wraps the lines in smaller resolution:

![image](https://user-images.githubusercontent.com/16364053/202424192-d00a5185-6fed-4521-96cc-738a422e8f0b.png)

It can be fixed by moving the Image into the last place in the DockPanel, but then the title moves during resizing...
The only solution I find is rolling back to `Grid`.
@jmacato I am not sure if wrapping the lines is a problem, what do you think? If you think that is not good the code below can fix it, if you don't then just ignore my comment.

```xml
        <Grid RowDefinitions="2*,3*,*">
          <TextBlock Grid.Row="0" Text="Welcome to Wasabi Wallet!" Classes="title1" DockPanel.Dock="Top" HorizontalAlignment="Center" VerticalAlignment="Center" />
          <DockPanel Grid.Row="1" LastChildFill="True" DockPanel.Dock="Bottom" VerticalAlignment="Center">
            <!-- Text -->
            <StackPanel VerticalAlignment="Center">
              <TextBlock Classes="title2">
                Bitcoin wallet
              </TextBlock>
              <TextBlock Classes="text">
                Receive and send money, securely and privately.
              </TextBlock>
              <TextBlock Classes="title2">
                Non-custodial
              </TextBlock>
              <TextBlock Classes="text">
                You are in full control, nobody can steal from you.
              </TextBlock>
              <TextBlock Classes="title2">
                Private by default
              </TextBlock>
              <TextBlock Classes="text">
                Because your transactions are nobody else's business.
              </TextBlock>
              <TextBlock Classes="title2">
                Open Source
              </TextBlock>
              <TextBlock Classes="text">
                Verify what’s going on and help us improve.
              </TextBlock>
            </StackPanel>

            <!-- Image -->
            <Panel DockPanel.Dock="Right"
                   MaxWidth="230"
                   VerticalAlignment="Center"
                   HorizontalAlignment="Right">
              <ExperimentalAcrylicBorder Margin="2">
                <ExperimentalAcrylicBorder.Material>
                  <ExperimentalAcrylicMaterial BackgroundSource="Digger"
                                               TintColor="White"
                                               TintOpacity="0"
                                               FallbackColor="White"
                                               MaterialOpacity="0" />
                </ExperimentalAcrylicBorder.Material>
              </ExperimentalAcrylicBorder>
              <Viewbox>
                <Panel>
                  <Border Width="465.5" Height="465.5" />
                  <Path Fill="{DynamicResource TextControlForeground}" Opacity="0.75"
                        Data="M 14 37 L 14 452 L 453 452 L 453 189 L 322 189 L 322 37 L 14 37 z " />
                  <Path Fill="#FF77C200" Opacity="0.75" Data="M 322 18 H 458 V 189 H 322 Z" />
                  <Path Fill="{DynamicResource RegionColor}"
                        Data="M0 0v465.5h465.5V0H0zm363.6 76.1h74.6v73.3h-74.6V76zM91.1 81.6a479.5 479.5 0 0 0 54.2 142.3l6.8 9.5h84a474.6 474.6 0 0 1-43.3-115.2L255 82.9a528 528 0 0 0 55.6 141l6.8 9.5h119.3v58.3h-65c19 20.4 40.6 35.3 59.6 47.5l-51.5 50.2c-25.8-17.7-70.5-50.2-107.1-97.7H209c20.4 17.7 44.8 39.4 58.3 47.5l-54.2 50.2A430 430 0 0 1 96.5 274.1a481 481 0 0 1-69.2-156c23-13.5 40.7-21.6 63.8-36.5z" />
                </Panel>
              </Viewbox>
            </Panel>
          </DockPanel>
        </Grid>
```

Reply:
It should wrap the text @soosr. Kinda pointless if the user doesnt see the full text at least when resized to that size